src/TransformServiceProvider.php

```diff
@@ -20,8 +20,16 @@ public function register()
 
     protected function registerTransform()
     {
-        $this->app->singleton('transform', function ($app) {
-            return new TransformService(new SimpleJsonSerializer());
+        $serializer = $this->getJsonSerializer();
```

Comment:
I would add a stock config file so that it installs in apps using `vendor:publish` (https://laravel.com/docs/5.3/packages). You can take a look at how `BugsnagServiceProvider` does this.

Reply:
Done